sdk/modelsrepository/Azure.Iot.ModelsRepository/README.md

```diff
@@ -0,0 +1,89 @@
+# Azure IoT Models Repository client library for .NET
+
+This library provides functionality for interacting with the Azure IoT Models Repository. It also aims to provide a consistent experience working with digital twin model repositories following Azure IoT conventions.
+
+[Source code][source] | [Package (nuget)][package]
```

Comment:
Source code link is currently pointing to master, which is fine, we just have to make sure the link is valid once we go to master (Analyze step during the pipeline usually confirms links)

Reply:
roger